.env.example

```diff
@@ -1,3 +0,0 @@
-PORT=9000
```

Comment:
let's rename then .env.example to .env.app or something else "example" - bad name